packages/taquito-beacon-wallet/test/taquito-beacon-wallet.spec.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  BeaconWallet,
+  BeaconWalletNotInitialized,
+  MissingRequiredScopes,
+} from '../src/taquito-beacon-wallet';
+import LocalStorageMock from './mock-local-storage';
+import { PermissionScope, LocalStorage, windowRef } from '@airgap/beacon-dapp';
+import sinon from 'sinon';
+
+global.localStorage = new LocalStorageMock();
+
+describe('Beacon Wallet tests', () => {
+
+  beforeEach(() => {
+    sinon.restore()
```

Comment:
Is `sinon` needed to avoid getting the following error?: `It looks like you created multiple Beacon SDK Client instances. This can lead to problems. Only create one instance and re-use it everywhere.`

Reply:
Sinon is not needed and i have removed it